cli/azd/pkg/azdext/extension_host.go

```diff
@@ -65,6 +78,9 @@ func NewExtensionHost(client *AzdClient) *ExtensionHost {
 		newServiceTargetManager: func(c *AzdClient) serviceTargetRegistrar {
 			return NewServiceTargetManager(c)
 		},
+		newFrameworkServiceManager: func(c *AzdClient) frameworkServiceRegistrar {
+			return NewFrameworkServiceManager(c)
+		},
```

Comment:
Any reason we can't just return the actual constructor function instead of wrapping in another func with the same args?

Reply:
followed what it was there for Service target